src/NodeJS/Server.php

```diff
@@ -528,4 +512,33 @@ protected function createTemporaryServer()
      * @return string The server's JavaScript code
      */
     abstract protected function getServerScript();
+
+    private function createProcess()
+    {
+        $arguments = array(
+            $this->nodeBin,
+            $this->serverPath,
+        );
+        $env = array(
+            'HOST' => $this->host,
+            'PORT' => $this->port,
+        );
+
+        if (!empty($this->nodeModulesPath)) {
+            $env['NODE_PATH'] = $this->nodeModulesPath;
+        }
+
+        if (!empty($this->options)) {
+            $env['OPTIONS'] = json_encode($this->options);
+        }
+
+        $process = new Process($arguments, null, $env);
```

Comment:
This will break with older versions of `symfony/process` which don't support passing an array of arguments (ProcessBuilder must still be used for them)

Reply:
Any suggestions how I can check the verison of `symfony/process`?

Reply:
check whether the method `fromShellCommandline` exists on the Process class. It was added when deprecating ProcessBuilder. If the method does not exist, it is an older version and ProcessBuilder should be used.

Reply:
Unfortunately this won't work. The method `fromShellCommandline` does not exist in v3.4